src/pages/pricing.tsx

```diff
@@ -123,36 +121,22 @@ const PricingPage: FunctionComponent = () => (
                     </div>
 
                     <div className="row justify-content-center pt-md-4">
-                        <div className="col-lg-10">
-                            <Link href="/case-studies/lyft-monolith-to-microservices" passHref={true}>
-                                {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                                <a className="pricing-page__quote-link">
-                                    <blockquote className="blockquote text-dark text-center">
-                                        <p className="text-left font-weight-bold border-left border-3 border-vermillion px-4">
-                                            &ldquo;Sourcegraph gives us the ability to search for and refactor
-                                            references to deprecated services, libraries, URL patterns, and more across
-                                            our 2000+ repositories, and the confidence that we're not leaving anyone
-                                            behind.&rdquo;
-                                        </p>
-                                        <small className="text-muted">
-                                            &mdash; Aneesh Agrawal, Software Engineer, Lyft
-                                        </small>
-                                        <img
-                                            src="/external-logos/lyft-logo.svg"
-                                            className="d-block mx-auto my-4"
-                                            width="87px"
-                                            alt="Lyft logo"
-                                        />
-                                    </blockquote>
-                                </a>
-                            </Link>
+                        <div className="col-md-8 text-center">
+                            <BlockquoteWithBorder
+                                quote="Sourcegraph gives us the ability to search for and refactor
+                                references to deprecated services, libraries, URL patterns, and more across
+                                our 2000+ repositories, and the confidence that we're not leaving anyone
+                                behind."
+                                author="Aneesh Agrawal, Software Engineer, Lyft"
+                                logo={{src: '/external-logos/lyft-logo.svg', alt: 'Lyft logo: Read the case study', href: '/case-studies/criteo-tackles-big-code'}}
+                            />
                         </div>
                     </div>
 
                     <div className="row pt-4">
                         <div className="col-md-12 mx-auto mb-4">
-                            <div className="pricing-plan pricing-plan--team card">
-                                <h2 className="card-title mt-3 mb-3 pricing-plan__title">Team</h2>
+                            <div className="px-6 card">
```

Comment:
It looks like the styling on "Team" section on `/pricing` is misaligned with the rest of the page. The text is smaller than comparable text on the page, and there appears to be some extra vertical padding on mobile. Perhaps we can match the style of the "Free" and "Enterprise" sections by using something like `text-xl` and `px-5 px-md-6`?

<img width="364" alt="Screen Shot 2022-07-06 at 11 05 00 AM" src="https://user-images.githubusercontent.com/60713139/177594711-d997bfc4-d411-47ee-9ca3-adc41c708f0d.png">

Reply:
Good eye! I anticipate this page will undergo a refresh/standardization soon so I don't want to spend too much time on it. But I did align these sections to have more similar spacing in [758e671](https://github.com/sourcegraph/about/pull/5556/commits/758e671aec8c5e8eadc8ffea4ce898d5659e986a) commit. Take another look! The tags are nested a bit weird, so I hope to clean up the template more later.